target/linux/ramips/rt288x/config-4.4

```diff
@@ -119,8 +119,8 @@ CONFIG_MIPS_CMDLINE_FROM_DTB=y
 CONFIG_MIPS_L1_CACHE_SHIFT=4
 CONFIG_MIPS_L1_CACHE_SHIFT_4=y
 # CONFIG_MIPS_MACHINE is not set
-CONFIG_MIPS_NO_APPENDED_DTB=y
-# CONFIG_MIPS_RAW_APPENDED_DTB is not set
+# CONFIG_MIPS_NO_APPENDED_DTB is not set
+CONFIG_MIPS_RAW_APPENDED_DTB=y
```

Comment:
As far as I have seen it does not create any issues to use both concurrently. Removing "CONFIG_MIPS_RAW_APPENDED_DTB=y" results in no DTB to be found if append-dtb and DEVICE_DTS is configured. I'm not aware if it is possible to set kernel configuration settings per board.